pages/property-filter/common-props.tsx

```diff
@@ -178,19 +180,57 @@ export const i18nStrings: PropertyFilterProps.I18nStrings & I18nStringsExt = {
   tokenLimitShowFewer: 'Show fewer',
   clearFiltersText: 'Clear filters',
   tokenOperatorAriaLabel: 'Boolean Operator',
-  removeTokenButtonAriaLabel: () => 'Remove token',
   enteredTextLabel: (text: string) => `Use: "${text}"`,
 
-  tokenEditorTokenActionsLabel: token =>
-    `Filter remove actions for ${token.propertyLabel} ${token.operator} ${token.value}`,
-  tokenEditorTokenRemoveLabel: () => 'Remove filter',
-  tokenEditorTokenRemoveFromGroupLabel: () => 'Remove filter from group',
+  formatToken,
+  removeTokenButtonAriaLabel: token => `Remove token, ${formatToken(token)}`,
+
+  groupEditAriaLabel: group => `Edit group with ${group.tokens.length} tokens`,
+  tokenEditorTokenActionsAriaLabel: token => `Filter remove actions for ${formatToken(token)}`,
+  tokenEditorTokenRemoveAriaLabel: token => `Remove filter, ${formatToken(token)}`,
+  tokenEditorTokenRemoveLabel: 'Remove filter',
+  tokenEditorTokenRemoveFromGroupLabel: 'Remove filter from group',
   tokenEditorAddNewTokenLabel: 'Add new filter',
-  tokenEditorAddTokenActionsLabel: 'Add filter actions',
-  tokenEditorAddExistingTokenLabel: token =>
-    `Add filter ${token.propertyLabel} ${token.operator} ${token.value} to group`,
+  tokenEditorAddTokenActionsAriaLabel: 'Add filter actions',
+  tokenEditorAddExistingTokenAriaLabel: token => `Add filter ${formatToken(token)} to group`,
+  tokenEditorAddExistingTokenLabel: token => `Add filter ${getTokenLabel(token)} to group`,
 };
 
+function getTokenLabel(token: PropertyFilterProps.FormattedToken) {
+  return `${token.propertyLabel} ${token.operator} ${token.value}`;
+}
+
```

Comment:
The getTokenLabel is used for visible label and formatToken is used for ARIA labels. The diff is in the operator presentation: for ARIA we use description instead of symbolic operator value.

Reply:
All that is only needed when using explicit i18nStrings, not relying on the built-in i18n.